starboard/evergreen/shared/launcher.py

```diff
@@ -271,6 +275,43 @@ def _StageTargetsAndContentsRaspi(self):
     os.makedirs(os.path.join(target_staging_dir, 'lib'))
     shutil.copy(target_binary_src, target_binary_dst)
 
+  def _StageTargetsAndContentsRdk(self):
+    """Stage targets and their contents for GN builds for RDK platforms."""
+    content_subdir = os.path.join('usr', 'share', 'cobalt')
+
+    # TODO(b/296633713):
+    # Target_name should be self.target_name. However because loader is
+    # hardcoded on RDK, i.e. it always loads "libcobalt.so" in path "cobalt",
+    # it has to use this name.
+    target_name = 'cobalt'
+
+    # Copy target content and binary.
+    target_install_path = os.path.join(self.out_directory, 'install')
+    target_staging_dir = os.path.join(self.staging_directory, 'content', 'app',
+                                      target_name)
```

Comment:
Does this actually work? The content path I think would not change due to the hard coded path in the loader.

Should the hardcoding of `target_name` happen below this line instead?

Reply:
This is to form the local staging folder, which will be mirrored/rsync to RDK. The "cobalt" is needed because the path is hardcoded as well.